src/DotVVM.Framework/Controls/UpdateProgress.cs

```
@@ -21,6 +36,11 @@ protected override void AddAttributesToRender(IHtmlWriter writer, IDotvvmRequest
         {
             writer.AddKnockoutDataBind("dotvvm-UpdateProgress-Visible", "true");
 
+            if (Delay != 0)
```

We should throw exception when invalid (negative number) value is entered.